tests/antithesis/docker-compose.yml

```diff
@@ -30,8 +33,11 @@ services:
       ETCD_INITIAL_CLUSTER_TOKEN: "etcd-cluster-1"
       ETCD_INITIAL_CLUSTER: "etcd0=http://etcd0:2380,etcd1=http://etcd1:2380,etcd2=http://etcd2:2380"
       ETCD_INITIAL_CLUSTER_STATE: "new"
+      ETCD_DATA_DIR: "/var/etcd/data"
     ports:
       - 22379:2379
+    volumes:
+      - ${ETCD_ROBUSTNESS_DATA_PATH-/tmp/etcddata}1:/var/etcd/data
```

Comment:
By default docker will mount it as root user. In local mode this will lead to `open /tmp/etcddata1/member/wal: permission denied` permission. Maybe we could mount it with same same user as caller? or just turn of support in local mode.

Reply:
Yep, that's what I found as well.  I think I just fixed it, but things are a bit more complicated in `Makefile` now.